gytheio-transform/gytheio-transform-commons/src/main/java/org/gytheio/content/transform/AbstractFileContentTransformerWorker.java

```diff
@@ -186,6 +186,7 @@ protected boolean isTempTargetUsed()
                     target = targetContentReferenceHandler.createContentReference(
                             resultFile.getName(), resultMediaType);
                 }
+                target.setSize(resultFile.length());
```

Comment:
LGTM - thanks for the unit test.

Minor comment: the same code on line 198 looks superfluous now (since already covered in both if / else ) ?